### ISSUE TITLE
startmapd fails in docker: missing libcuda.so.1

### ISSUE THREAD

**Issue by nagautier** [omnisci/mapd-core#281]
Running the latest version of mapd/mapd-ce-cuda fails with the following error message:
`./bin/initdb: error while loading shared libraries: libcuda.so.1: cannot open shared object file: No such file or directory`
This is running the startmapd manually in a container. I'm using nvidia-docker2.

**Comment by andrewseidl** [comment]
Make sure you're passing the runtime parameter when starting the container, otherwise the driver and libraries won't be available inside the container:

    docker run --runtime=nvidia mapd/mapd-ce-cuda

**Comment by nagautier** [comment]
Hi! 
thanks for the quick response, some more details:
I'm running cuda 10.0 on the host machine (driver version is 410). OS is ubuntu18.04. I believe the docker image is running cuda 9.2 with ubuntu16.04 though. What I've tried:
docker run \
  --runtime=nvidia \
  -d \
  --name mapd \
  -p 9091-9092:9091-9092 \
  -v /home/username/mapd-storage:/mapd-storage \
  mapd/mapd-ce-cuda
This yields an ID but no container is actually alive, I assume since the CMD in the image is:
 /mapd/startmapd --non-interactive --data /mapd-storage/data --config /mapd-storage/mapd.conf
My second try was to run the image in interactive mode with a bash endpoint:
docker run \
  --runtime=nvidia \
  -it \
  --name mapd \
  -p 9091-9092:9091-9092 \
  -v /home/username/mapd-storage:/mapd-storage \
  mapd/mapd-ce-cuda bash
And then running:
 /mapd/startmapd --non-interactive --data /mapd-storage/data --config /mapd-storage/mapd.conf
Which is how I got the error. I've since tried installing the cuda drivers on the host machine, which I believe is should not be required and it now seems to be working.
I'm excited about trying this out!
